docs/manifests/kubent-cronjob.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can we add `Namespace` resource for `kubent-system` too? I realise that you expect the Prometheus push gateway anyway, but I think it would be nice to have this working out of the box (I believe `apply` would fail if the namespace is not present currently).